API Google Analytics/HelloAnalytics.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+
+from apiclient.discovery import build
+from oauth2client.service_account import ServiceAccountCredentials
+
+import httplib2
+from oauth2client import client
+from oauth2client import file
+from oauth2client import tools
+
+SCOPES = ['https://www.googleapis.com/auth/analytics.readonly']
+DISCOVERY_URI = ('https://analyticsreporting.googleapis.com/$discovery/rest')
+KEY_FILE_LOCATION = 'client_secrets.p12'
+SERVICE_ACCOUNT_EMAIL = ''
+VIEW_ID = ''
+
+def initialize_analyticsreporting():
+  """Initializes an analyticsreporting service object.
+
+  Returns:
+    analytics an authorized analyticsreporting service object.
+  """
+
+  credentials = ServiceAccountCredentials.from_p12_keyfile(
+    SERVICE_ACCOUNT_EMAIL, KEY_FILE_LOCATION, scopes=SCOPES)
+
+  http = credentials.authorize(httplib2.Http())
+
+  # Build the service object.
+  analytics = build('analytics', 'v4', http=http, discoveryServiceUrl=DISCOVERY_URI)
+
+  return analytics
+
```

Comment:
This function looks like a exception generator. What do you think about we use try, except and raise statements.